15_perms_lab/permslab.txt

```diff
@@ -1,15 +1,52 @@
 1. Using the file structure in the current directory, write the commands that perform the following:
 
   - change permission of file src/main.c to r--r--r-- using chmod letter notation
+chmod 444 src/main.c
+
 
   - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
 
+chmod 564 include/player.h
+[note: for the group ALL should be the order in r-- ? Like rwx?]
+
   - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
 
+[q: is it possible to undo? Sadly I made a mistake an changed also permissions of Doc folder itself]
+
+chmod 600 Doc/*.* 
+[I changed again the permisson and after running the above command I see that the permisson only the file now is as requsted, My reasoning was that only if it's a file we need to change and this is why I used *.*]. 
```

Comment:
In future lessons we'll learn about `find` command which will give you the fine grained control you need to actually run chmod on files only